### DISCUSSION TITLE
Google Slide Show

### DISCUSSION THREAD

**Discussion by RuthAnn1** [Techwise-Team-15/my_pet#37] [General]
Here is the link to the slide show: https://docs.google.com/presentation/d/19mIUaIFJaa84h4ZJPVkD5hbuRitDjryfPm_fjDFRIWY/edit?usp=sharing

Let me know if you have any trouble accessing the slides :)

**Comment by dinaolmelak** [comment]
@RuthAnn1 I see this
<img width="687" alt="Screenshot 2023-07-10 at 1 14 19 PM" src="https://github.com/Techwise-Team-15/my_pet/assets/32272045/875592d4-4eb6-49ff-8fcc-ad236eb410ea">

**Reply by RuthAnn1** [reply]
click request access and it will email me an approval. @dinaolmelak

**Reply by dinaolmelak** [reply]
✅ Done
